src/signature_pad.ts

```diff
@@ -138,6 +138,7 @@ export default class SignaturePad {
         callback(error);
       }
     };
+    image.crossOrigin = 'Anonymous';
```

Comment:
Not sure if it matters but the spec shows it as lowercase. [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)
```suggestion
    image.crossOrigin = 'anonymous';
```